docs/rules/valid-title.md

```diff
@@ -148,3 +148,45 @@ describe('foo', () => {
   test('bar', () => {});
 });
 ```
+
+## Options
+
+```ts
+interface {
+  ignoreTypeOfDescribeName?: boolean;
+  disallowedWords?: string[];
+}
+```
+
+#### `ignoreTypeOfDescribeName`
+
+Default: `false`
+
+When enabled, the type of the first argument to `describe` blocks won't be
+checked.
+
+#### `disallowedWords`
+
+Default: `[]`
+
+A string array of words that are not allowed to be used in test titles. Matching
+is not case-sensitive, and looks for complete words:
+
+Examples of **incorrect** code using `disallowedWords`:
+
+```js
+// with disallowedWords: ['correct', 'all', 'every', 'properly']
+describe('the correct way to do things', function() {});
```

Comment:
use arrow functions here as well to keep them similar except for what the rule enforces?